tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/AutoProjectDeletionSpec.scala

```diff
@@ -0,0 +1,44 @@
+package ch.epfl.bluebrain.nexus.tests.kg
+
+import akka.http.scaladsl.model.StatusCodes
+import ch.epfl.bluebrain.nexus.tests.BaseSpec
+import ch.epfl.bluebrain.nexus.tests.Identity.projects.Bojack
+import ch.epfl.bluebrain.nexus.tests.iam.types.Permission.{Events, Organizations, Projects, Resources}
+import io.circe.Json
+
+import scala.concurrent.duration._
+
+/**
+  * Test related to automatic project deletion Only checks that the project deletion has been triggered and that the
+  * project itself has been deleted. All the additional checks on the deletion of resources/views contained in this
+  * project are done in [[ProjectsDeletionSpec]]
+  *
+  * @see
+  *   ProjectsDeletionSpec
+  */
```

Comment:
It's probably worth mentioning the configuration that this test relies on ("autodeletion" is in the configuration as an organisation which will auto delete projects after 5 seconds)